requirements/prod-requirements.txt

```diff
@@ -192,7 +192,7 @@ eulxml==1.1.3
     # via -r base-requirements.in
 executing==0.8.3
     # via stack-data
-flower==1.2.0
+flower @ git+https://github.com/mher/flower@a4fc7c97d897a3172d1058d5269115eb88d811de
```

Comment:
Here's the diff for anyone who wants to look over it. Two-way diff: https://github.com/mher/flower/compare/v1.2.0..a4fc7c97d897a3172d1058d5269115eb88d811de. One-way diff https://github.com/mher/flower/compare/v1.2.0...a4fc7c97d897a3172d1058d5269115eb88d811de.

Looks fine to me